source/winAPI/sessionTracking.py

```diff
@@ -155,74 +122,37 @@ def isWindowsLocked() -> bool:
 	Not to be confused with the Windows sign-in screen, a secure screen.
 	"""
 	from core import _TrackNVDAInitialization
+	from winAPI.sessionTracking import _isWindowsLocked_checkViaSessionQuery
+	from systemUtils import _isSecureDesktop
 	if not _TrackNVDAInitialization.isInitializationComplete():
-		# During NVDA initialization,
-		# core._initializeObjectCaches needs to cache the desktop object,
-		# regardless of lock state.
-		# Security checks may cause the desktop object to not be set if NVDA starts on the lock screen.
-		# As such, during initialization, NVDA should behave as if Windows is unlocked.
 		return False
 	if _isSecureDesktop():
-		# If this is the Secure Desktop,
-		# we are in secure mode and on a secure screen,
-		# e.g. on the sign-in screen.
-		# _isSecureDesktop may also return True on the lock screen before a user has signed in.
-
-		# For more information, refer to devDocs/technicalDesignOverview.md 'Logging in secure mode'
-		# and the following userGuide sections:
-		# - SystemWideParameters (information on the serviceDebug parameter)
-		# - SecureMode and SecureScreens
 		return False
-	lockStateTracked = _hasLockStateBeenTracked()
-	if lockStateTracked:
-		return WindowsTrackedSession.SESSION_LOCK in _currentSessionStates
+	global _initializationLockState

```

Comment:
is `_initializationLockState` used for anything?

Reply:
The intention is explained here: https://github.com/nvaccess/nvda/pull/14416/files#r1042835284, I'll add a code comment.